common/structs/src/fee.rs

```diff
@@ -16,6 +19,42 @@ pub enum FeeType<M: ManagedTypeApi> {
     },
 }
 
+#[type_abi]
+#[derive(TopEncode, TopDecode, NestedDecode, Clone)]
+pub struct AddUsersToWhitelistOperation<M: ManagedTypeApi> {
+    pub nonce: TxNonce,
+    pub users: ManagedVec<M, ManagedAddress<M>>,
+}
+
+impl<A: CryptoApi> GenerateHash<A> for AddUsersToWhitelistOperation<A> {}
+
+#[type_abi]
+#[derive(TopEncode, TopDecode, NestedDecode, Clone)]
+pub struct RemoveUsersFromWhitelistOperation<M: ManagedTypeApi> {
+    pub nonce: TxNonce,
+    pub users: ManagedVec<M, ManagedAddress<M>>,
+}
+
+impl<A: CryptoApi> GenerateHash<A> for RemoveUsersFromWhitelistOperation<A> {}
+
+#[type_abi]
+#[derive(TopDecode, TopEncode, NestedEncode, NestedDecode, Clone)]
+pub struct RemoveFeeOperation<M: ManagedTypeApi> {
+    pub nonce: TxNonce,
+    pub token_id: TokenIdentifier<M>,
+}
+
+impl<A: CryptoApi> GenerateHash<A> for RemoveFeeOperation<A> {}
+
+#[type_abi]
+#[derive(TopDecode, TopEncode, NestedEncode, NestedDecode, Clone)]
+pub struct SetFeeOperation<M: ManagedTypeApi> {
+    pub nonce: TxNonce,
+    pub fee_struct: FeeStruct<M>,
```

Comment:
Let's be consistent with execute operation.
Put struct first and nonce after.

```suggestion
    pub fee_struct: FeeStruct<M>,
    pub nonce: TxNonce,
```

Reply:
Same for all operations